offix/src/main/java/org/aerogear/offix/interfaces/ConflictResolutionImpl.kt

```diff
@@ -0,0 +1,12 @@
+package org.aerogear.offix.interfaces
+
+/*
+ Users can implement ConflictResolutionImpl interface in their app and provide a custom conflict resolution implementation.
+ */
+interface ConflictResolutionImpl {
```

Comment:
Call this `ConfliceResolutionInterface`
Impl means you are providing the implementation details in this class which we are not.